ecosystem/sep-0031.md

```diff
@@ -187,9 +188,10 @@ This requires the Receiving Anchor to implement the [SEP-38 Anchor RFQ API](sep-
 1. The Sending Anchor makes [SEP-12 PUT /customer](sep-0012.md#customer-put) requests containing the collected information for each client.
 1. If the Receiving Anchor supports the [SEP-38 Anchor RFQ API](sep-0038.md), the Sending Anchor can request a quote from the Receiving Anchor. See the [Receiving Anchor Asset Conversion](#receiving-anchor-asset-conversion) section for more information.
 1. The Sending Anchor makes a `POST /transactions` request to create a transaction record with the Receiving Anchor. This request contains the `id`s returned by the `PUT /customer` requests, the `transaction.fields` values collected from the Sending Client, as well as the transaction information. The response will include a `id` that will be used to check the transaction data and status.
-1. The Sending Anchor makes `GET /transactions/:id` requests until the transaction's `status` is `pending_sender`.
-1. The Sending Anchor submits the payment transaction to Stellar using the information provided in the `GET /transactions/:id` response.
-1. The Sending Anchor makes `GET /transactions/:id` requests until the transaction's `status` is `completed`, `error`, `pending_customer_info_update`, or `pending_transaction_info_update`.
+1. Optionally, the Sending Anchor makes a `PUT /transactions/:id/callback` request to register a URL that the Receiving Anchor will make requests to when the transaction's status changes.
+1. The Sending Anchor makes a `GET /transactions/:id` request to ensure the status is `pending_sender`. If it is not, the Sending Anchor must wait until it receives a callback request or poll the `GET /transactions/:id` endpoint until the status becomes `pending_sender`.
+1. The Sending Anchor submits the payment transaction to Stellar using the information provided in the `POST /transactions` response.
```

Comment:
This wording contradicts what we agreed for PR [#1203](https://github.com/stellar/stellar-protocol/pull/1203/files#diff-6abf4f55bde3bfb0f1b81e2437e70d45ea84c64213c3d09715bd901e3ec2a101R191). 

The wallet should actually double-check the amount_in from the GET /transactions/:id before sending the payment.